src/Nuget.PackageIndex.Manager/PackageIndexManager.cs

```diff
@@ -72,7 +71,7 @@ private void ProcessAction(ILocalPackageIndexBuilder builder, Arguments argument
             switch (arguments.Action)
             {
                 case PackageIndexManagerAction.Build:
-                    result = builder.BuildAsync().Result;
+                    result = builder.BuildAsync(newOnly:false, cancellationToken: CancellationToken.None).Result;
```

Comment:
you always want to auto format before saving. This will prevent unnecessary changes due to formatting on the next commit